src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListServiceImpl.java

```diff
@@ -59,13 +60,16 @@ public PostDailyListResponse saveDailyListToDatabase(DailyListPostRequest postRe
             //update the record
             savedDailyListEntity = existingRecordOpt.get();
             dailyListMapper.updateDailyListEntity(postRequest, listingCourthouse, savedDailyListEntity);
+            savedDailyListEntity.setLastModifiedBy(userIdentity.getUserAccount());
```

Comment:
every time we call `userIdentity.getUserAccount()` there are all sorts of lookups on the database, so if we need to call it, probably best to just call it once and keep the result.

Reply:
fixed